### PR TITLE
Add openPMD files created from the code FEMM

### PR DESCRIPTION
**Pull request by RemiLehe**
This PR adds:
- Two new openPMD files, which were created with the code FEMM : `example-femm-3d.h5` and `example-femm-thetaMode.h5`
- A new folder `scripts/femm` that contains the scripts that show how these files were created.

To when this PR, I would suggest first having a look at:
- `README.md`
- `scripts/femm/README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,20 @@ cd openPMD-example-datasets
...
 tar -zxvf example-3d.tar.gz
 tar -zxvf example-thetaMode.tar.gz
+...
 ```
 
-Note: The datasets were produced with the Particle-In-Cell code
-[Warp](https://bitbucket.org/berkeleylab/warp), from the scripts in
-`scripts/`. The resolution is intentionally very low, so as to produce
-data of manageable size.
+# How where these files created?
```
Comment (Yin-YinjianZhao) [inline README.md]: How were

File: README.md [hunk 2]
```diff
@@ -16,9 +16,20 @@ cd openPMD-example-datasets
...
+## Files created with the Particle-In-Cell code Warp
+
+The files `example-2d.tar.gz`, `example-3d.tar.gz` and `example-thetaMode.tar.gz` were produced with the Particle-In-Cell code
+[Warp](https://bitbucket.org/berkeleylab/warp). The resolution is intentionally very low, so as to produce data of manageable size.
+
+The scripts that were used in order to produce these openPMD files from Warp can be found in the directory `scripts/warp`.
+
+## Examples created with the finite-element code FEMM
+
+The files `example-femm-3d.h5` and `example-femm-thetaMode.h5` were produced withe the finite-element code [FEMM](https://www.femm.info/wiki/HomePage).
```
Comment (Yin-YinjianZhao) [inline README.md]: were produced within

**Review by Yin-YinjianZhao — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (RemiLehe) [inline README.md]:
```suggestion
The files `example-femm-3d.h5` and `example-femm-thetaMode.h5` were produced using the finite-element code [FEMM](https://www.femm.info/wiki/HomePage).
```

File: README.md (hunk 1 above)
Comment (RemiLehe) [inline README.md]:
```suggestion
# How were these files created?
```

File: scripts/femm/readme.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+`mirror.FEM` can be loaded by the FEMM software
```
Comment (ax3l) [inline scripts/femm/readme.txt]: Awesome! Can you please combine this with the `README.md` in the same dir?
That will just render nicer when someone opens the dir on GitHub.

Can you note the FEMM version that was used, so we remember in a few years?

File: scripts/femm/readme.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+`openpmd_create_3d.py` and `openpmd_create_rz.py`
+are python scripts to convert `B*_3d.txt` and `B*_rz.txt`
+to openPMD data files, named `openpmd_data_3d.h5` and
+`openpmd_data_rz.h5`, respectively.
+
+`openpmd_data_3d.h5` is used in WarpX example `LoadExternalField3D`;
+`openpmd_data_rz.h5` is used in WarpX example `LoadExternalFieldRZ`.
+
+The WarpX input files of these two example tests are in
+`Examples/Tests/LoadExternalField/`.
```
Comment (ax3l) [inline scripts/femm/readme.txt]: Can you please add a `requirements.txt` file here that documents the code versions used for the scripts?
Check this out:
https://github.com/ComputationalRadiationPhysics/particle_reduction/blob/f4d008bfd5f82192b0b3c3bb4d4ad7469637d492/requirements.txt#L3

(also add numpy pls)

File: scripts/femm/readme.txt (hunk 3 above)
Comment (ax3l) [reply scripts/femm/readme.txt]: @RemiLehe I pushed a commit that fixes that. Can you pls take a look?

**Review by ax3l — APPROVED** [review]

File: scripts/femm/readme.txt (hunk 4 above)
Comment (ax3l) [reply scripts/femm/readme.txt]: pushed it :)

**Pull request merged**